src/variety/isowatari.js

```diff
@@ -0,0 +1,494 @@
+//
+// isowatari.js
+//
+
+(function(pidlist, classbase) {
+	if (typeof module === "object" && module.exports) {
+		module.exports = [pidlist, classbase];
+	} else {
+		pzpr.classmgr.makeCustom(pidlist, classbase);
+	}
+})(["isowatari"], {
+	MouseEvent: {
+		use: true,
+		inputModes: {
+			edit: ["circle-shade", "circle-unshade", "empty", "clear"],
+			play: ["shade", "unshade", "clear"]
+		},
+
+		inputEdit: function() {
+			// 初回はこの中に入ってきます。
+			if (this.inputData === null) {
+				this.inputEdit_first();
+			} else {
+				this.inputqnum_main(this.getcell());
+			}
+			// 境界線の入力中の場合
+		},
+		inputEdit_first: function() {
+			var bd = this.board,
+				bx = this.inputPoint.bx,
+				by = this.inputPoint.by,
+				rect = bd.clusterSize.rect;
+			if (
+				bx >= rect.bx1 &&
+				bx <= rect.bx2 &&
+				by >= rect.by1 &&
+				by <= rect.by2
+			) {
+				var val = this.getNewNumber(bd.clusterSize, bd.clusterSize.count);
+				if (val === null) {
+					this.inputqnum_main(this.getcell());
+				}
+				bd.clusterSize.set(val);
+				this.mousereset();
+			}
+			// その他は境界線の入力へ
+			else {
+				this.inputqnum_main(this.getcell());
+			}
+		},
+
+		mouseinput_auto: function() {
+			if (this.puzzle.playmode) {
+				if (this.mousestart || this.mousemove) {
+					this.inputcell();
+				}
+				if (this.notInputted() && this.mousestart) {
+					this.inputqcmp();
```

Comment:
This if-statement can be removed, since marking cells as completed is not applicable to this genre (and is causing tests to fail, currently)